src/app/pages/ParaTimesPage/TransactionAmount/index.tsx

```diff
@@ -0,0 +1,231 @@
+                            }
+                          : undefined,
+                      (amount: string) =>
+                        isAmountGreaterThan(amount, balance)
+                          ? {
+                              message: t('errors.insufficientBalance', 'Insufficient balance'),
+                              status: 'error',
+                            }
+                          : undefined,
+                      (amount: string) =>
+                        isAmountGreaterThan(
+                          amount,
+                          getMaxAmountValue(
+                            isDepositing,
+                            balance,
+                            transactionForm.feeAmount,
+                            balanceInBaseUnit,
+                            consensusDecimals,
+                            decimals,
+                          ),
+                        )
+                          ? {
+                              message: t(
+                                'paraTimes.validation.insufficientBalanceToPayFee',
+                                'Insufficient balance to pay the fee',
+                              ),
+                              status: 'error',
+                            }
+                          : undefined,
+                    ]}
+                  >
+                    <TextInput
+                      disabled={disabled}
+                      inputMode="decimal"
+                      name="amount"
+                      placeholder="0"
+                      value={transactionForm.amount}
+                    />
+                  </FormField>
+
+                  <Box
+                    style={{
+                      fontWeight: 'bold',
+                      zIndex: 2,
+                      position: 'absolute',
+                      top: '15px',
+                      right: 0,
+                    }}
+                  >
+                    <Tip
+                      content={t('paraTimes.amount.tooltip', 'Max value may be decreased by the fee')}
+                      dropProps={{ align: { bottom: 'top' } }}
+                    >
+                      <Button
+                        disabled={disabled}
+                        plain
+                        label={t('paraTimes.amount.max', 'MAX')}
+                        onClick={() =>
+                          setTransactionForm({
+                            ...transactionForm,
+                            amount: formatBaseUnitsAsRose(
+                              getMaxAmountValue(
+                                isDepositing,
+                                balance,
+                                transactionForm.feeAmount,
+                                balanceInBaseUnit,
+                                consensusDecimals,
+                                decimals,
+                              ),
+                            ).replaceAll(',', ''),
```

Comment:
Now that there's no conditional `formatter`, can this be refactored without `replaceAll(',', '')`?